### ISSUE TITLE
TabsBar: Incorrect semantics and keyboard behavior if slotted anchor tags are used

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2040]
Scope
---
If Tabs Bar is used with **anchor links** the semantics and keyboard behaviour should be adapted. 

Out of scope
---
- Tabs component is not affected
- No separate component, stick with tabsbar

Note
---
- links should be navigable by tab key only (not with arrow keys)
- the semantics should follow standard navigation guidelines, see WCAG best practices
- add validation for supported nesting and correct usage of ONLY <a> OR <button> (no mixing)

Also the documentation should clarify exactly those different usage scenarios.

Out of scope
---
- No handling of mixed `a` and `button` elements

Acceptance Criteria
---
- [x] TabsBar with nested links is implemented accessible compliant and with expected interaction
- [x] Validation is implemented
